src/js/crypto.cpp

```diff
@@ -597,6 +597,8 @@ namespace ccf::js
           {key, key + key_size},
           label_opt);
 
+        OPENSSL_cleanse(wrapping_key, wrapping_key_size);
```

Comment:
The wrapping key is public in this case, so there is no benefit to limiting its lifetime on the heap. Also this scrubs the contents of the input array buffer directly, (see line 557), as opposed to the one backing the temporary Pem object created on line 596, so the user may be surprised and annoyed if they try to re-use it.